views/page/index.haml

```diff
@@ -11,5 +11,7 @@
     - pages.each do |page|
       %li
         %a{ href: "#{page.slug}" }= page.title
+        - if page.concealed
+          🔐
```

Comment:
Mostly to make it easier when playing around with this, WDYT, keep? @dentarg

Reply:
Sure, some emoji is always nice